DEPLOYMENT_PROCEDURE.md

```diff
@@ -4,23 +4,26 @@ This document is only intended for Pathoplexus sysadmins.
 
 ## To develop new versions of Pathoplexus
 
-Work in a development branch. You can choose what version of Loculus to target by adjusting the `loculusVersion` in  `pathoplexus_app/values.yaml`. Use a Loculus commit SHA.
+Work in a development branch. You can choose what version of Loculus to target by adjusting the `loculusVersion` in  `pathoplexus_app/values.yaml` (use a Loculus commit SHA). Adding the `preview` label will create a preview.
 
-## Merging to main
+## To simply bump the version of Loculus used in Pathoplexus `main`
+We have a GitHub workflow for this. Go to the "Actions" tab -> Open the "Create PR to bump main pathoplexus.." action -> Open the "Run workflow" dropdown -> Click on the "Run workflow" button. This will create a PR, which someone else will need to approve. The PR will contain a changelist for Loculus, which you should carefully review for breaking changes.
 
-Open a PR to merge into main. The current main is visible at http://preview-main.pathoplexus.org.
+## Staging the latest main
 
-## Staging a new version
+Once you have new release candidate on main, you can point the staging server at it by changing the `head_sha` value in at https://github.com/pathoplexus/loculus_deployments/blob/main/deploy/staging/config.json .
 
-Once you have new release candidate you can point the staging server at it by changing the head_sha in the `staging` folder in https://github.com/pathoplexus/loculus_deployments/tree/main/deploy .
+The easiest way to do this is with the GitHub action "Create PR to bump staging to latest" in the `loculus_deployments` repository: https://github.com/pathoplexus/loculus_deployments/actions
 
-The changes will be reflected at http://demo.pathoplexus.org . You should also check the health of the application in ArgoCD.
+A little after the PR is merged, changes will be reflected at http://demo.pathoplexus.org . You should also check the health of the application in ArgoCD.
```

Comment:
We might want to add a commit sha somewhere on the website (through config) to make it easier to check that the deployment was successful

Reply:
We might want to do so, but checking this wouldn't be sufficient: it would only tell you about one pod

Reply:
For sure not sufficient, just necessary